fxapom/pages/marionette/sign_in.py

```diff
@@ -0,0 +1,86 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, Wait, By
+
+from fxapom.fxapom import TIMEOUT
+from fxapom.pages.marionette.base import Base
+
+
+class MarionetteSignIn(Base):
+
+    _fox_logo_locator = (By.ID, 'fox-logo')
+    _email_input_locator = (By.CSS_SELECTOR, '.input-row .email')
+    _marionette_email_input_locator = (By.CSS_SELECTOR, '.input-row .email')
+    _next_button_locator = (By.ID, 'email-button')
+    _password_input_locator = (By.ID, 'password')
+    _sign_in_locator = (By.ID, 'submit-btn')
+
+    def __init__(self, driver, timeout=TIMEOUT):
+        super(Base, self).__init__(driver, timeout)
+        self._sign_in_window_handle = None
+        self.popup = None
+        self.driver = driver
+        self.timeout = timeout
```

Comment:
These two lines duplicate what's already done by the parent class and so can be removed.